alg/gdaltransformer.cpp

```diff
@@ -1669,6 +1669,26 @@ bool GDALComputeAreaOfInterest(OGRSpatialReference* poSRS,
  * GeoTIFF datasets should be used to store the backmap. The default is NO, that
  * is to use in-memory arrays, unless the number of pixels of the geolocation
  * array is greater than 16 megapixels.
+ * <li> GEOLOC_ARRAY/SRC_GEOLOC_ARRAY=filename. (GDAL &gt;= 3.5.2) Name of a
+ * GDAL dataset that contains at least 2 bands with the X and Y geolocation bands.
+ * This is an alternative to having geolocation information described in the
+ * GEOLOCATION metadata domain of the source dataset. The geolocation array dataset
+ * specified this way may have a GEOLOCATION metadata domain (its X_DATASET and
+ * Y_DATASET metadata items may be missing and will be ignored), or it might be
+ * completely missing, in which case the geolocation array should have only 2
+ * bands, the first one will be used as the X geolocation band and the second one
+ * as the Y geolocation band. If the geolocation dataset has a SRS set, it will
```

Comment:
@rouault agreed definitely tedious. Maybe something like this (omitted formatting to make review easier)?

```
GEOLOC_ARRAY/SRC_GEOLOC_ARRAY=filename. (GDAL &gt;= 3.5.2)

Name of a GDAL dataset containing a geolocation array and associated metadata.
This is an alternative to having geolocation information described in the
GEOLOCATION metadata domain of the source dataset. The dataset specified may
have a GEOLOCATION metadata domain containing appropriate metadata, however
default values are assigned for all omitted items. X_BAND defaults to 1 and
Y_BAND to 2, however the dataset must contain exactly 2 bands. PIXEL_OFFSET
and LINE_OFFSET default to 0. PIXEL_STEP and LINE_STEP default to 1. SRS defaults
to the geolocation array dataset's spatial reference system if set, otherwise
WGS84 is used. GEOREFERENCING_CONVENTION is selected from the main metadata
domain if it is omitted from the GEOLOCATION domain, and if not available
TOP_LEFT_CORNER is assigned as a default. If GEOLOC_ARRAY is set SRC_METHOD
defaults to GEOLOC_ARRAY.


DST_GEOLOC_ARRAY=filename. (GDAL &gt;= 3.5.2)

See SRC_GEOLOC_ARRAY for details, assumptions, and defaults. If DST_GEOLOC_ARRAY
is set DST_METHOD defaults to GEOLOC_ARRAY.
```

Reply:
thanks!